Sources/DocCDocumentation/DocCDocumentation.docc/V5_API_Migration_guide.md

```diff
@@ -0,0 +1,82 @@
+
+The SDK will automatically use StoreKit 1 in the following versions where StoreKit 2 is not supported: on macOS 11 or earlier, iOS 14 or earlier, iPadOS 14 or earlier, tvOS 14 or earlier, or watchOS 7 or earlier.
+
+If for any reason you need to always use StoreKit 1, it is possible to switch back using the following configuration API:
+
+```swift
+Purchases.configure(with: .builder(withAPIKey: apiKey)
+    .with(storeKitVersion: .storeKit1)
+    .build()
+```
+
+### 3rd Party Analytics SDKs
+
+If you are using any 3rd party analytics SDKs to automatically track in-app purchases, you need to be aware most of them do not completely support logging purchases made with StoreKit 2. This is the case for some popular SDKs like Facebook, Mixpanel, OneSignal, Segment or Firebase. For these services, we recommend you use our [data integrations](https://www.revenuecat.com/integrations/).
+
+If you're using the Firebase SDK, you'll need to follow [these instructions](https://firebase.google.com/docs/analytics/measure-in-app-purchases#swift) to log purchases made with StoreKit 2.
+
+### Observer Mode
+
+Version 5.0 of the SDK introduces support for observer mode when making purchases with StoreKit 2. You can enable it when configuring the SDK:
+
+```swift
+Purchases.configure(with: .builder(withAPIKey: apiKey)
+		.with(observerMode: true, storeKitVersion: .storeKit2)
+    .build()
+```
+
+Additionally, because of the way in which StoreKit 2 operates, you must manually handle newly purchased transactions by calling `Purchases.shared.handleObserverModeTransaction()`. Subscription updates will be automatically tracked.
+
+```swift
+let result = try await product.purchase()
+Purchases.shared.handleObserverModeTransaction(result)
+```
+
+If you're using observer mode with StoreKit 1, you will need to explicitly configure the SDK to use StoreKit 1:
+
+```swift
+Purchases.configure(with: .builder(withAPIKey: apiKey)
+		.with(observerMode: true, storeKitVersion: .storeKit1)
+    .build()
+```
+
+### Original Application Version
+
+If you're converting a paid app to in-app subscriptions, and want to provide existing customers with certain features, [we recommend](https://www.revenuecat.com/blog/engineering/converting-a-paid-ios-app-to-subscriptions/) using the "original app version" field in customer info.
+
+Because of a limitation of StoreKit 2, this field is not available if the customer is running on iOS 15, tvOS 15, macOS 12 or watchOS 8. If this is a requirement for you, we recommend switching back to StoreKit 1.
+
+## Trusted Entitlements
+
+Version 5.0 of the SDK enables the Informational mode for Trusted Entitlements by default to protect your purchases from attackers. See the [Trusted Entitlements documentation](https://www.revenuecat.com/docs/trusted-entitlements) for more information.
```

Comment:
I would rephrase this a bit since by default it doesn't actually "protect" you unless you check anything.
Maybe mention that informational will just log verification errors and allow you to check `customer info.entitlements.verification`.